static/js/controllers/inbox.js

```diff
@@ -378,43 +382,30 @@ var feedback = require('../modules/feedback'),
           render: function() {
             $scope.openTourSelect();
             var id = 'org.couchdb.user:' + Session.userCtx().name;
-            UpdateUser(id, { known: true })
-              .catch(function(err) {
-                $log.error('Error updating user', err);
-              });
+            UpdateUser(id, { known: true }).catch(function(err) {
+              $log.error('Error updating user', err);
+            });
           }
         },
       ];
 
-      var filteredModals;
-      var modalsInited = {
-        guidedSetup: false,
-        welcome: true,
-        userLanguage: true
-      };
-
-      var showModals = function() {
-        if (filteredModals && _.every(_.values(modalsInited))) {
-          // render the first modal and recursively show the rest
-          if (filteredModals.length) {
-            filteredModals.shift().render(function() {
-              showModals(filteredModals);
-            });
-          }
-        }
-      };
-
-      DB()
-        .query('medic-client/doc_by_type', { key: [ 'translations', true ] })
-        .then(function(result) {
-          $scope.enabledLocales = _.pluck(result.rows, 'value');
-        });
+      Languages().then(function(languages) {
+        $scope.enabledLocales = languages;
```

Comment:
can we chuck that out of the `inbox` parent scope? Looks like we're part of the way there, a couple modals have their own copy. 
(don't like untrackable parent scope vars!)